src/viewer/main/VisItViewer.C

```diff
@@ -93,6 +93,13 @@ VisItViewer::Initialize(int *argc, char ***argv)
             debug5 << "Viewer launching with OSPRay" << endl;
             avtCallback::SetUseOSPRay(true);
         }
+#endif
+#ifdef VISIT_ANARI
+        else if (strcmp((*argv)[i], "-anari") == 0)
```

Comment:
Does adding `-anari` to the command line update the RenderingAttributes `Anari` checkbox?
Can it only be enabled on startup, or is choosing the option via the RenderingAttributes sufficient?
Just wondering why the need for the command line option?

Reply:
Not needed, I removed this. I was just copying OSPRay.